content/docs/1.10.0/nodes-and-volumes/nodes/multidisk.md

```diff
@@ -5,47 +5,111 @@ weight: 4
+1. Choose a Disk: Select the physical or virtual disk you want to use for Longhorn storage and format it with an extent-based filesystem (e.g., ext4, xfs).
+2. Mount the Disk: Mount the disk to a directory on the host, such as `/mnt/example-disk`. Ensure the directory is accessible and properly configured.
 
-Longhorn will detect the storage information (e.g. maximum space, available space) about the disk automatically, and start scheduling to it if it's possible to accommodate the volume. A path mounted by the existing disk won't be allowed.
+Then, you can add the disk using either the Longhorn UI or the `kubectl` command-line tool. Choose the method that suits your preference.
 
-A certain amount of disk space can be reserved to stop Longhorn from using it. It can be set in the `Space Reserved` field for the disk. It's useful for the non-dedicated storage disk on the node.
+- **Using the Longhorn UI**
 
-The kubelet needs to preserve node stability when available compute resources are low. This is especially important when dealing with incompressible compute resources, such as memory or disk space. If such resources are exhausted, nodes become unstable. To avoid kubelet `Disk pressure` issue after scheduling several volumes, by default, Longhorn reserved 30% of root disk space (`/var/lib/longhorn`). This default value is defined by the [storage-reserved-percentage-for-default-disk](../../../references/settings/#storage-reserved-percentage-for-default-disk) setting to help ensure node stability.
+  1. Head to the `Node` tab, select one of the nodes, and select `Edit Disks` in the dropdown menu.
+  2. Add the path of the mounted disk into the disk list of the node.
 
-> **Note**:
-> Since Longhorn uses filesystem ID to detect duplicate mounts of the same filesystem, you cannot add a disk that has the same filesystem ID as an existing disk on the same node.
-> See more details at https://github.com/longhorn/longhorn/issues/2477
+- **Using `kubectl` Command**
+
+  1. Execute `kubectl edit node.longhorn.io <node-name>` to edit the node resource.
+  2. Add the disk path to `spec.disks` in the node resource. For example:
+
+    ```yaml
+    ...
+    spec:
+      ...
+      disks:
+        ...
+        example-disk:
+          allowScheduling: true
+          diskDriver: ""
+          diskType: filesystem
+          evictionRequested: false
+          path: /mnt/example-disk
+          storageReserved: 0
+          tags: []
+    ...
+    ```
+
+  3. Save and exit the editor.
+
+Once a disk is added:
+
+- Longhorn automatically detects the storage details of the disk, such as maximum and available capacity.
+- If the disk is suitable for storing volume data, Longhorn begins scheduling volumes to it.
+
+Optionally, you can use the `Space Reserved` field in the UI or `spec.disks.<disk-name>.storageReserved` to reserve a portion of disk space (in bytes) for other purposes. This reserved space will not be used by Longhorn for volume data.
```

Comment:
Consider having this to `### Rook Disk Reservation`.

Reply:
Good idea. Added, thank you.